message.rb

```diff
@@ -0,0 +1,50 @@
+def decode(morse)
+    morse_code = {
+        'A' => '.-' ,
+        'B' => '-...' ,
+        'C' => '-.-.' ,
+        'D' => '-..' ,
+        'E' => '.' ,
+        'F' => '..-.' ,
+        'G' => '--.' ,
+        'H' => '....' ,
+        'I' => '..' ,
+        'J' => '.---' ,
+        'K' => '-.-' ,
+        'L' => '.-..' ,
+        'M' => '--' ,
+        'N' => '-.' ,
+        'O' => '---' ,
+        'P' => '.--.' ,
+        'Q' => '--.-' ,
+        'R' => '.-.' ,
+        'S' => '...' ,
+        'T' => '-' ,
+        'U' => '..-' ,
+        'V' => '...-' ,
+        'W' => '.--' ,
+        'X' => '-..-' ,
+        'Y' => '-.--' ,
+        'Z' => '--..' ,
+        '1' => '.----' ,
+        '2' => '..---' ,
+        '3' => '...--' ,
+        '4' => '....-' ,
+        '5' => '.....' ,
+        '6' => '-....' ,
+        '7' => '--...' ,
+        '8' => '---..' ,
+        '9' => '----.' ,
+        '0' => '-----' ,
+        ' ' => '/' 
+    }
+
+morse = morse.gsub!("   ", " / ", )
+
+morse.split(' ').each do |el|
+        print morse_code.key(el).to_s
+end (edited) 
```

Comment:
- [ ] Could you remove `(edited)` it's causing your program to crash when we run it!
```suggestion
end 
```